### ISSUE TITLE
[BUG] [fd error]: Problem while executing command: Argument list too long (os error 7) when using "-l"

### ISSUE THREAD

**Issue by sebma** [sharkdp/fd#1179]
### Checks

- [X] I have read the troubleshooting section and still think this is a bug.

### Describe the bug you encountered:

Hi, I get this error `[fd error]: Problem while executing command: Argument list too long (os error 7)` when using `-l` : 

```shell
$ cd /mnt/Netapp/files/_plugins/fusioninventory/files/repository
$ fdfind -t f -S +1Mi -l
[fd error]: Problem while executing command: Argument list too long (os error 7)
$ fdfind -t f -S +1Mi | wc -l
16701
$
```

### Describe what you expected to happen:

List the files without error.

### What version of `fd` are you using?

fd 8.3.1

### Which operating system / distribution are you on?

```shell
$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 22.04.1 LTS
Release:        22.04
Codename:       jammy
$
```

**Comment by sharkdp** [comment]
Fixed since 8.4.0 (https://github.com/sharkdp/fd/releases/tag/v8.4.0)

> "Argument list too long" errors can not appear anymore when using --exec-batch/-X, as the command invocations are automatically batched at the maximum possible size, even if --batch-size is not given. See https://github.com/sharkdp/fd/issues/410 and https://github.com/sharkdp/fd/pull/1020 (@tavianator)

**Comment by sebma** [comment]
@tmccombs Thanks a lot. I downloaded and installed `fdfind v8.4.0` from https://packages.ubuntu.com/kinetic/fd-find and now it works fine :

```shell
$ fdfind -V
fd 8.4.0
$ fdfind -t f -S +1Mi -l | wc -l
16701
$
```
